.github/workflows/release_candidate.yml

```diff
@@ -69,3 +70,13 @@ jobs:
               description: package.version,
               target_url: `https://unpkg.com/${package.name}@${package.version}/`
             })
+
+  rc-at-dotcom:
+      name: Run primer integration workflow at dotcom
+      needs: release-candidate
+      # Using the branch name for ref as the PR is still WIP - TODO: Remove this once we confirm that the workflow is running as expected
+      uses: github/github/.github/workflows/primer-react-nightly.yml@primer-react-at-RC-time
+      # TODO: FInd out how to get the PR number for the release and the sha that is merged into main and trigger the workflow with those inputs 
+      with:
+        merged-pr: 'TODO: Get the SHA that is merged into main and triggers the workflow'
+        release-pr: 'TODO: Get the PR number for the release'
```

Comment:
Curious to hear what you think about passing this type of info to [the primer workflow at dotcom ](https://github.com/github/github/blob/d2468742a1d0e84b29769f34147d589502da0680/.github/workflows/primer-react-nightly.yml) 

`merged-pr`: This could be the PR number or commit SHA of the PR that is merged to main and triggers this release_candidate workflow. If we can pass this info to [the primer workflow at dotcom ](https://github.com/github/github/blob/d2468742a1d0e84b29769f34147d589502da0680/.github/workflows/primer-react-nightly.yml) we could use this associated commit SHA or the PR number on [automatically created integration PR commit messages](https://github.com/github/github/pull/259780/commits) 

For example  `Integration tests for @primer/react "35.20.0-rc.8c5c51d2 - merged commit SHA: <commit-sha>` . so that we can easily see which PR or commit SHA is associated with that package version on the commit message to debug easier. 

`release-pr`: https://github.com/primer/react/pulls/primer-css We can maybe link this to the body or title of [the automatically created integration PR](https://github.com/github/github/pull/259780/commits)

Reply:
@broccolinisoup is there a good overview of the expected flow for the PRs generated from this? I think depending on that the answers I would give would change 👀

In terms of metadata, it'd be great to have some structure similar to dependabot like: 

- PR title: `chore(dependencies): update @primer/react to vA.B.C`
- Commit title: `chore(dependencies): update @primer/react to vA.B.C`
- Message body:
  - Include a link to a commit, if possible, for the release
  - Include a changelog, if possible, for the release


Separately, for an ideal flow I would love it if we could have dotcom and PRC be as tightly aligned as possible so the intent of these workflows would be to:

- Create a PR against dotcom with the latest version at least every day
- If all is green, merge the PR
- If anything fails, a comment is posted in Slack for the release conductor

Reply:
Will this run the nightly workflow on the github/github repo? I think this will try to run the `primer-react-nightly.yml` workflow in the primer/react repo